packages/next/taskfile.js

```diff
@@ -2418,13 +2418,5 @@ export async function release(task) {
 
 // notification helper
 function notify(msg) {
```

Comment:
You can remove this function and all calls to it in this case. Also the `notifier` dependency should be removed 👍